lib/api/controllers/serverController.js

```diff
@@ -130,29 +154,31 @@ function ServerController (kuzzle) {
       }
     });
 
-    response.kuzzle.plugins = kuzzle.pluginsManager.getPluginsFeatures();
+    response.kuzzle.plugins = this.kuzzle.pluginsManager.getPluginsFeatures();
 
-    return Object.keys(kuzzle.services.list).reduce((previous, current) => {
+    let serviceName;
+
+    return Object.keys(this.kuzzle.services.list).reduce((previous, current) => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 81 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)